code/modules/vehicles/cars/car.dm

```diff
@@ -93,3 +96,117 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedastrian focused station, and now all these cars are ruining it!." //ok boomer
```

Comment:
```suggestion
	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
```
i know this is for april fools so shittiness is part and parcel but... also boomers don't care about the environment

Reply:
back in their days everyone rode horses